registration/tests/test_views.py

```diff
@@ -123,10 +122,10 @@ def test_reg_source_session(self):
         session = self.client.session
 
         for key in (
-                SESSION_EMAIL, SESSION_STATE, SESSION_STATE_NAME, SESSION_POSTAL_CODE, SESSION_REGISTRATION_CONFIGURATION,
-                SESSION_ACCEPTS_REGISTRATION, SESSION_REDIRECT_URL, SESSION_REGISTRATION_UPDATE, SESSION_COBRAND_COMPANY_NAME,
-                SESSION_COBRAND_COMPANY_LOGO, SESSION_COBRAND_ACTIVE, SESSION_COBRAND_ID, SESSION_WIDGET_HOST_URL,
-                SESSION_WIDGET_ID,):
+            SESSION_EMAIL, SESSION_STATE, SESSION_STATE_NAME, SESSION_POSTAL_CODE, SESSION_REGISTRATION_CONFIGURATION,
+            SESSION_ACCEPTS_REGISTRATION, SESSION_REDIRECT_URL, SESSION_REGISTRATION_UPDATE, SESSION_COBRAND_COMPANY_NAME,
+            SESSION_COBRAND_COMPANY_LOGO, SESSION_COBRAND_ACTIVE, SESSION_COBRAND_ID,
+                SESSION_WIDGET_HOST_URL, SESSION_WIDGET_ID,):
```

Comment:
@waseem-omar  No this violate the following PEP8 standard  `E125 continuation line with same indent as next logical line`
If we add to the same line we will break the  rules of limit line length